web/components/input/buttons/CheckboxGroup/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const InputContext = React.createContext();
+
+const Checkbox = ({ name, value, label, checked, disabled, onChange }) => (
+  <InputContext.Consumer>
```

Comment:
This is pretty cool, never seen this pattern before.

Would you still be able to render a `Checkbox` without `CheckboxGroup`?

Reply:
IKR, it's the [new context API](https://reactjs.org/docs/context.html) and I like it a lot. :smiley: 

I intended for `Checkbox` to be used on its own but currently it's failing without a parent `CheckboxGroup`. I'll fix this tomorrow, thanks!